docs/zh_cn/user_guides/4_pretrain_custom_dataset.md

```diff
@@ -1,29 +1,29 @@
-# Tutorial 4: Pretrain with Custom Dataset
+# 教程 4: 使用自定义数据集进行预训练
 
-- [Tutorial 4: Pretrain with Custom Dataset](#tutorial-4-pretrain-with-custom-dataset)
-  - [Train MAE on Custom Dataset](#train-mae-on-custom-dataset)
-    - [Get the path of custom dataset](#get-the-path-of-custom-dataset)
-    - [Choose one config as template](#choose-one-config-as-template)
-    - [Edit the dataset related config](#edit-the-dataset-related-config)
-  - [Train MAE on COCO Dataset](#train-mae-on-coco-dataset)
-  - [Train SimCLR on Custom Dataset](#train-simclr-on-custom-dataset)
-  - [Load Pre-trained Model to Speedup Convergence](#load-pre-trained-model-to-speedup-convergence)
+- [教程 4: 使用自定义数据集进行预训练](#教程-4-使用自定义数据集进行预训练)
+  - [在自定义数据集上使用MAE算法进行预训练](#在自定义数据集上使用mae算法进行预训练)
+    - [第一步：获取自定义数据路径](#第一步获取自定义数据路径)
+    - [第二步：选择一个配置文件作为模板](#第二步选择一个配置文件作为模板)
+    - [第三步：修改数据集相关的配置](#第三步修改数据集相关的配置)
+  - [在COCO数据集上使用MAE算法进行预训练](#在coco数据集上使用mae算法进行预训练)
+  - [在自定义数据集上使用SimCLR算法进行预训练](#在自定义数据集上使用simclr算法进行预训练)
+  - [使用MMSelfSup提供的预训练模型来加速收敛](#使用mmselfsup提供的预训练模型来加速收敛)
 
-In this tutorial, we provide some tips on how to conduct self-supervised learning on your own dataset(without the need of label).
+在本教程中，我们将介绍如何使用自定义数据集(无需标注)进行自监督预训练。
 
-## Train MAE on Custom Dataset
+## 在自定义数据集上使用MAE算法进行预训练
```

Comment:
```suggestion
## 在自定义数据集上使用 MAE 算法进行预训练
```
Better to add space between Chinese character and English letter to make it more clear and readable, please check other parts of the doc